### ISSUE TITLE
[1.5.5 -> master] too-many-function-args false positives

### ISSUE THREAD

**Issue by The-Compiler** [PyCQA/pylint#926]
I noticed a couple of issues when trying out astroid/pylint master, with a codebase which is pylint-clean with astroid 1.4.5 and pylint 1.5.5. I'll open an issue for thing I think is separate.

Those can all be reproduced by cloning the [qutebrowser repository](https://github.com/The-Compiler/qutebrowser) and running `tox -e pylint-master -r`. All line numbers are on commit 24db93f3eb17b97e8928d5faf26dd56221030493 of qutebrowser.

--- 

```
************* Module qutebrowser.misc.earlyinit
E:207,13: Too many positional arguments for method call (too-many-function-args)

************* Module qutebrowser.mainwindow.statusbar.bar
E:350, 8: Too many positional arguments for method call (too-many-function-args)
E:358,12: Too many positional arguments for method call (too-many-function-args)
E:375, 8: Too many positional arguments for method call (too-many-function-args)
E:454,12: Too many positional arguments for method call (too-many-function-args)
E:468,12: Too many positional arguments for method call (too-many-function-args)
E:513, 8: Too many positional arguments for method call (too-many-function-args)
E:536,16: Too many positional arguments for method call (too-many-function-args)
```

The one in `earlyinit` is a simple `sys.exit` call:

```python
def check_pyqt_core():
    """Check if PyQt core is installed."""
    try:
        import PyQt5.QtCore
    except ImportError as e:
        # [...]
        if '--debug' in sys.argv or '--no-err-windows' in sys.argv:
            print(file=sys.stderr)
            traceback.print_exc()
        sys.exit(1)   # <---
```

The others are all `self.txt.set_text` calls in [bar.py](https://github.com/The-Compiler/qutebrowser/blob/master/qutebrowser/mainwindow/statusbar/bar.py).

Example:

```python
class StatusBar(QWidget):

    # [...]

    def __init__(self, win_id, parent=None):
        # [...]

        self.txt = textwidget.Text()

        # [...]

    # [...]

    def _set_mode_text(self, mode):
        """Set the mode text."""
        text = "-- {} MODE --".format(mode.upper())
        self.txt.set_text(self.txt.Text.normal, text)   # <---
```

``textwidget.Text` looks like this:

```python
class Text(textbase.TextBase):

    # [...]

    def set_text(self, which, text):
        # [...]
```

**Comment by PCManticore** [comment]
Fixed by astroid's latest commit, let me know if you can still reproduce it.
